### PR TITLE
Ha azure aws

### PR DESCRIPTION
**Pull request by DevOpsDerek**

### CODE REVIEW

**Review by harrisonmeister — APPROVED** [review]
LGTM. Waiting :crossed_fingers: for an editorial review

File: docs/administration/high-availability/design/octopus-for-high-availability-on-aws.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+For the sake of simplicity, this guide assumes that all of the servers used for your Octopus High Availability instance are hosted in AWS.
+
+:::warning
+If you are choosing [IaaS](https://en.wikipedia.org/wiki/Infrastructure_as_a_service) on AWS then the [On-Premises](/docs/administration/high-availability/design/octopus-for-high-availability-on-premises.md) guide might be a better approach for you as you may have your Domain Controllers, SQL Database Server and load balancers in the cloud.
+:::
+
+### Compute
+
+To install Octopus nodes you will need at least to machines running Windows Server 2016 +. There’s only one choice when building virtual machines in AWS, and that’s [EC2 Instances](https://aws.amazon.com/ec2/instance-types/). There are a number of different instance types to choose from. When selecting the size of the instance, We generally find sticking with the General purpose size is the best option. Choosing the rights specs for the instance depends on how many nodes you plan to use.
```
Comment (pstephenson02) [inline docs/administration/high-availability/design/octopus-for-high-availability-on-aws.md:25]: Minor typo here in the first sentence: `... you will need at least to machines` - two :)

File: docs/administration/high-availability/design/octopus-for-high-availability-on-aws.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+### Database
+
+Each Octopus Server node stores project, environment and deployment-related data in a shared Microsoft SQL Server Database. Since this database is shared, it's important that the database server is also highly available. 
+
+If you don't have a SQL cluster in AWS then AWS provides a fully managed and highly available sql database as a service called [Amazon RDS for SQL Server](https://aws.amazon.com/rds/sqlserver/).
+
+Choosing a SQL edition and size is dependent on your organization and Octopus usage. The great thing about using AWS RDS is that you can start small and scale on demand as your Octopus usage grows. For further information see our [SQL Server Database](/docs/installation/sql-server-database.md) page, which explains the editions and versions of SQL Server that Octopus supports and explains the requirements for how the database must be configured.
```
Comment (pstephenson02) [inline docs/administration/high-availability/design/octopus-for-high-availability-on-aws.md:33]: Scaling the size of course is an option, but I'm not sure that changing the edition is possible after you've created it. A little Googling led me to this: https://stackoverflow.com/a/60302666/15507446

Even the SQL Server Database installation page doesn't offer many considerations about which edition should be used. Does it make sense to offer a suggestion here? I think in the past I've suggested customers use Standard Edition.

File: docs/administration/high-availability/design/octopus-for-high-availability-on-aws.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+AWS recently introduced `Amazon FSx`, It’s a native Windows file system built on Windows Server. It includes full support for the SMB protocol, Windows NTFS, and Microsoft Active Directory (AD) integration and is an ideal choice for connecting to your EC2 instances hosting Octopus to store all your Octopus packages and log files.
+
+If you choose to go with Amazon FSx there are some resources that will help you get started:
+- AWS have a [starter guide](https://docs.aws.amazon.com/fsx/latest/WindowsGuide/getting-started.html) which explains how to configure Amazon FSx and connect it up to an EC2 machine.
+- We have an [AWS FSx High Availability blog post](https://octopus.com/blog/aws-fsx-ha) which is a step-by-step guide to connecting up Amazon FSx to your Octopus High Availability Server nodes on Windows.
+
+### Load Balancing in AWS
+
+To distibute traffic to the Octopus web portal on multiple nodes then you will need to use a HTTP loadbalancer. AWS provides a solution to distribute HTTP/HTTPS traffic to EC2 instances, Elastic Load Balancing is a highly available, secure, and elastic load balancer. There are three implementations of ELB;
```
Comment (pstephenson02) [inline docs/administration/high-availability/design/octopus-for-high-availability-on-aws.md:51]: distibute --> distribute

File: docs/administration/high-availability/design/octopus-for-high-availability-on-aws.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+* [Application Load Lalancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/introduction.html)
```
Comment (pstephenson02) [inline docs/administration/high-availability/design/octopus-for-high-availability-on-aws.md:53]: typo: Balancer

File: docs/administration/high-availability/design/octopus-for-high-availability-on-aws.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+* [Network Load Balancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/introduction.html)
+* [Classic Load Balancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/introduction.html)
+
+If you are using [Listening Tentacles](/docs/infrastructure/deployment-targets/windows-targets/tentacle-communication.md#polling-tentacles) then we recommend using the Application Load Balancer.
```
Comment (pstephenson02) [inline docs/administration/high-availability/design/octopus-for-high-availability-on-aws.md:57]: The link text refers to listening tentacles, but the actual link goes to polling 🐙

File: docs/administration/high-availability/design/octopus-for-high-availability-on-aws.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+However, [Polling Tentacles](/docs/infrastructure/deployment-targets/windows-targets/tentacle-communication.md#listening-tentacles-recommended) don't work so well with the Application Load Balancer, so instead we recommend using the Network Load Balancer. To setup a Network Load Balancer for Octopus High Availability with Polling Tentacles take a look at this [knowledge base article](https://help.octopus.com/t/how-can-i-configure-my-polling-tentacles-to-hit-my-octopus-deploy-high-availability-instance-to-sitting-behind-an-aws-load-balancer/24890).
```
Comment (pstephenson02) [inline docs/administration/high-availability/design/octopus-for-high-availability-on-aws.md:59]: Opposite here: link text says polling, but the link goes to listening.

**Review by pstephenson02 — COMMENTED** [review]
This is awesome guys, thanks first of all. I haven't read through all of it yet, but thought I should at least leave the few things I saw here to get fixed up.

**Review by wordlee — APPROVED** [review]
Minor line edits committed.

File: docs/administration/high-availability/design/octopus-for-high-availability-on-aws.md (hunk 2 above)
Comment (harrisonmeister) [reply docs/administration/high-availability/design/octopus-for-high-availability-on-aws.md]: Added recommendation, and re-worded slightly

**Pull request merged**